common/licenses/MIT

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
Unfortunately, every project seems to customize this line of the MIT license and so we can't really use a common copy of it, like we can for Apache-2.0 or MPL-2.0.

Did you actually find a project/library that had a copyright line with these placeholders? (I tried looking through the PR but the GitHub UI is unusable given the size)

Reply:
This is an example https://github.com/multiformats/rust-cid. There is no license file, and MIT is mentioned by passing at the bottom of the README.md, but the link to it is broken.

Reply:
Actually I've managed to find the actual license for these crates, added it to the crates.